src/uvw/emitter.hpp

```diff
@@ -178,7 +177,9 @@ class Emitter {
      */
     void clearAll() noexcept {
         for(auto &&h: handlers) {
-            h->clear();
+            if (h) {
```

Comment:
I'm doing a review of `emitter.hpp`, I'll add the patch to that commit. Thank you.